content/home/examples/a-stateful-component.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Stateful Component
+title: Durumlu Bir Bileşen
 order: 1
 domid: timer-example
 ---
 
-In addition to taking input data (accessed via `this.props`), a component can maintain internal state data (accessed via `this.state`). When a component's state data changes, the rendered markup will be updated by re-invoking `render()`.
+Bir bileşen, girdi verileri (`this.props` ile ulaşılabilir) almanın yanı sıra, dahili durum verilerini (`this.state` ile ulaşılabilir) yönetebilir. Bir bileşenin durum verileri değiştiğinde, oluşturulan görüntü `render()` methodunun tekrar çağırılması ile güncellenecektir.
```

Comment:
We decided to not translate `state` words, as you might know. So perhaps, `dahili state verisi` and `Bir bilesenin state verisi`. What do you think about these?

Reply:
I'll go with `dahili state verisi` then.